src/Driver/Driver.jl

```diff
@@ -202,6 +202,17 @@ struct SolverConfiguration{FT}
     solver
 end
 
+"""
+    DGmethods.courant(local_cfl, solver_config::SolverConfiguration;
+                      Q=solver_config.Q, dt=solver_config.dt)
+
+Returns the maximum of the evaluation of the function `local_courant`
+pointwise throughout the domain with the model defined by `solver_config`. The
+keyword arguments `Q` and `dt` can be used to call the courant method with a different state `Q` or time step `dt` than are defined in `solver_config`.
```

Comment:
shorten line length

Reply:
Not sure how that happened! How embarrassing...